app/src/main/java/com/example/songme/utils/ConvertJsonToObject.kt

```diff
@@ -0,0 +1,30 @@
+package com.example.songme.utils
+
+import com.example.songme.data.model.Track
+import org.json.JSONArray
+
+fun convertJsonToListTrack(jsonData: String): ArrayList<Track> {
+    val arrTrack = ArrayList<Track>()
+    val jsonArray = JSONArray(jsonData)
+    for (i in 0 until jsonArray.length()) {
+        val jsonObject = jsonArray.getJSONObject(i)
+        val jsonObjectUser = jsonObject.getJSONObject("user")
+        arrTrack.add(
+            Track(
+                jsonObject.getString("id"),
+                jsonObject.getString("kind"),
+                jsonObject.getString("created_at"),
+                jsonObject.getString("last_modified"),
+                jsonObject.getString("title"),
+                jsonObject.getString("duration").toLong(),
+                jsonObject.getString("stream_url"),
+                jsonObject.getString("uri"),
+                jsonObject.getString("artwork_url"),
+                jsonObject.getString("genre"),
+                jsonObject.getString("tag_list"),
+                jsonObjectUser.getString("username")
+            )
```

Comment:
Tạo thêm 1 constructor của Track với tham số truyền vào là jsonObject 
```suggestion
            Track(jsonObject)
```